### PR TITLE
Capi demo

### PR DESCRIPTION
**Pull request by aug24**
## What's changed?

I hate the capi filters
No, hang on, that hasn't changed.
You can now avoid the capi filters.

### General
- [ ] 🤖 Relevant tests added
- [ ] ✅ CI checks / tests run locally
- [ ] 🔍 Checked on CODE

### Client
- [ ] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [ ] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [ ] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

File: app/model/editions/templates/Capi.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  def build(c:Capi) = this.copy(next = Some(c))
+}
+
+object Capi{
+  def apply() = new Capi("?tag=")
+  def apply(s: String) = new Capi(s)
+  def article=Capi("type/article")
+  def lifeandstyle=Capi("lifeandstyle/lifeandstyle")
+
+  def features=Capi("tone/features")
```
Comment (juanvaccari) [inline app/model/editions/templates/Capi.scala:33]: I was wondering how do you reify the tags, or sections, etc. I see you are mapping them manually. But there are more than 50,000 tags. And they are mutating all the time, I guess? At least more are added frequently. So how would you handle that?

Comment (juanvaccari) [reply app/model/editions/templates/Capi.scala:33]: I must say I really like it, it is very elegant and safe to be able to write queries like this. I am just not sure about this practical issue.

Comment (aug24) [reply app/model/editions/templates/Capi.scala:33]: Whoever is writing the client will define the tags they want.  This section of code would not be in a real client.

Comment (aug24) [reply app/model/editions/templates/Capi.scala:33]: The `import model.editions.templates.Capi._` import at the top where the items are used is all that matters.

Comment (juanvaccari) [reply app/model/editions/templates/Capi.scala:33]: OK, so the client needs to define at compile time which tags they will use, which will work well if they use a limited and known set of them, for what I understand.

Comment (aug24) [reply app/model/editions/templates/Capi.scala:33]: You could easily make it dynamic.  Parse values out, wrap them into Capi class instances and include them.  That's what functional languages do best.

**Pull request closed without merging**